components/snowflake/snowflake.app.mjs

```diff
@@ -169,12 +172,12 @@ export default {
       const binds = [
         this.$auth.schema,
       ];
-      const rows = await this.collectRows({
+      const rows = await this.executeQuery({
         sqlText,
         binds,
       });
       return rows.reduce((acc, row) => {
-        acc[row.tableName] ??= {
+        acc[row.tableName] = {
```

Comment:
The `rows` list will have one entry per column, so it's very likely that there're multiple entries for a single table, like this:
```js
[
  {tableName: "users", columnName: "id", rowCount: 10, ...},
  {tableName: "users", columnName: "firstName", rowCount: 10, ...},
  {tableName: "users", columnName: "lastName", rowCount: 10, ...},
  {tableName: "users", columnName: "email", rowCount: 10, ...},
  {tableName: "pipelines", columnName: "id", rowCount: 200, ...},
  {tableName: "pipelines", columnName: "description", rowCount: 200, ...},
]
```

The `??=` operator prevents us from unnecessary building this metadata object and overriding the same entry multiple times.
```suggestion
        acc[row.tableName] ??= {
```